README.md

```diff
@@ -1,2 +1,67 @@
+[![CircleCI](https://circleci.com/gh/giantswarm/valuemodifier.svg?&style=shield&circle-token=2f991b43e9c0147771ebe0575885501a7184dea9)](https://circleci.com/gh/giantswarm/valuemodifier)
+
 # valuemodifier
-Package valuemodifier provides an interface to modify values of arbitrary structures in custom ways. 
+Package valuemodifier provides an interface to modify values of arbitrary
+structures in custom ways. Currently JSON and YAML formats are supported.
+
+### usage
+This is an working example of how to use this package. For more examples check
+the go tests.
+```
```

Comment:
AFAIK the go idiomatic way is to have examples as tests. There are tests. Ross requested to have an example in the readme. I am fine with this. Let me know when you have string feelings about an example directory. In go I am not used to that. Maybe I am missing something.